packages/postgres/README.md

```diff
@@ -142,6 +171,8 @@ As with the non-validated query functions, there are several variants available:
 - `callValidatedOneRow`
 - `callValidatedZeroOrOneRow`
```

Comment:
I'm curious: isn't there a version similar to `queryRow` et al for calling a sproc? Or is the fact that we're moving away from sprocs a consideration here?

Reply:
I was asking myself the same question when writing these docs! I think we avoided adding those functions exactly because we're trying to move away from sprocs; we don't want to make it easy/nice to use sprocs, or otherwise encourage their use in new code. That said, I could see making a pragmatic choice to introduce them in the future if that will make the conversion of existing code to TypeScript easier.